ui/perfherder/compare/CompareView.jsx

```diff
@@ -142,8 +142,19 @@ class CompareView extends React.PureComponent {
 
     tableNames.forEach((testName) => {
       rowNames.forEach((value) => {
-        if (!oldStddevVariance[value]) {
-          oldStddevVariance[value] = {
+        if (value.includes("base-rev")) {
+          return;
+        }
+
+        let base_value = value
+        if (origResultsMap.get(
```

Comment:
`origResultsMap` is created by calling the function `getResultsMap` - I'm thinking that maybe this logic can be moved there